lib/routes/api/service-providers.js

```diff
@@ -43,9 +62,9 @@ decorate(router, ServiceProvider, ['delete'], preconditions);
 
 router.use('/:sid/RecentCalls', hasServiceProviderPermissions, require('./recent-calls'));
 router.use('/:sid/Alerts', hasServiceProviderPermissions, require('./alerts'));
-router.use('/:sid/SpeechCredentials', hasServiceProviderPermissions, require('./speech-credentials'));
+router.use('/:sid/SpeechCredentials', hasAccountPermissions, require('./speech-credentials'));
 router.use('/:sid/Limits', hasServiceProviderPermissions, require('./limits'));
-router.use('/:sid/PredefinedCarriers', hasServiceProviderPermissions, require('./add-from-predefined-carrier'));
+router.use('/:sid/PredefinedCarriers', hasAccountPermissions, require('./add-from-predefined-carrier'));
```

Comment:
@davehorton I tried to let the account users to manage on the account level the PredefinedCarriers and SpeechCredentials. This is still prevented by the api. WHat else do I need to do here to allow this?